### ISSUE TITLE
Help on Profile Builder

### ISSUE THREAD

**Issue by Garpian** [laravel-enso/Enso#171]
This is  a **feature request**.

Is there or can you add a short guide on how to use the Profile Builder?

I like your framework very much, but i have trouble to discover the intended way to make the "show"(view) work for added resources.

Thank you very much.

**Comment by aocneanu** [comment]
Hi @Garpian,

I'm having troubles understanding your request.

Do you want to edit the existing ProfileBuilder? Or maybe you want to build your own show pages for other models?

Please be more specific.

P.S: I'm glad that you like Enso!

**Comment by Garpian** [comment]
Yes i want to build show pages for other models and thought that the ProfileBuilder for the user class is a template for that, because no other model has a show method.
But it seems that i have just to discover how to hand down laravel model objects to vue files.
Or is the intended way to always request data for the show view per axios?

**Comment by aocneanu** [comment]
A clean way to do that would be creating [resource](https://laravel.com/docs/5.7/eloquent-resources) classes and returning them from the controller.

You can see an example in the [teams index](https://github.com/laravel-enso/Teams/blob/master/src/app/Http/Controllers/TeamController.php#L14) page / TeamController@index method. Instead of returning `Resource::collection` you will return `new ModelResource($model)` for only one model

**Comment by Garpian** [comment]
Ok, i will look into this.
Thank you.

**Comment by aocneanu** [comment]
Can we close this?
